diff --git a/packages/client/components/ui/components/features/profiles/ProfileActions.tsx b/packages/client/components/ui/components/features/profiles/ProfileActions.tsx
index c8f3f34..b9a7bd6 100644
--- a/packages/client/components/ui/components/features/profiles/ProfileActions.tsx
+++ b/packages/client/components/ui/components/features/profiles/ProfileActions.tsx
@@ -2,6 +2,7 @@ import { Show } from "solid-js";
 
 import { useNavigate } from "@solidjs/router";
 import { ServerMember, User } from "stoat.js";
+import { Trans } from "@lingui-solid/solid/macro";
 import { styled } from "styled-system/jsx";
 
 import { UserContextMenu } from "@revolt/app";
@@ -50,11 +51,13 @@ export function ProfileActions(props: {
   return (
     <Actions width={props.width}>
       <Show when={props.user.relationship === "None" && !props.user.bot}>
-        <Button onPress={() => props.user.addFriend()}>Add Friend</Button>
+        <Button onPress={() => props.user.addFriend()}>
+          <Trans>Add Friend</Trans>
+        </Button>
       </Show>
       <Show when={props.user.relationship === "Incoming"}>
         <Button onPress={() => props.user.addFriend()}>
-          Accept friend request
+          <Trans>Accept friend request</Trans>
         </Button>
         <IconButton onPress={() => props.user.removeFriend()}>
           <MdCancel />
@@ -62,11 +65,13 @@ export function ProfileActions(props: {
       </Show>
       <Show when={props.user.relationship === "Outgoing"}>
         <Button onPress={() => props.user.removeFriend()}>
-          Cancel friend request
+          <Trans>Cancel friend request</Trans>
         </Button>
       </Show>
       <Show when={props.user.relationship === "Friend"}>
-        <Button onPress={openDm}>Message</Button>
+        <Button onPress={openDm}>
+          <Trans>Message</Trans>
+        </Button>
       </Show>
 
       <Show
