diff --git a/packages/client/components/rtc/index.ts b/packages/client/components/rtc/index.ts
index 6d1e346..5933af7 100644
--- a/packages/client/components/rtc/index.ts
+++ b/packages/client/components/rtc/index.ts
@@ -1,4 +1,4 @@
-export { VoiceContext, useVoice } from "./state";
+export { NOME_CANAL_AUSENTE, VoiceContext, useVoice } from "./state";
 
 export { InRoom } from "./components/InRoom";
 export { RoomAudioManager } from "./components/RoomAudioManager";
